src/freertos_drivers/common/WifiDefs.hxx

```diff
@@ -45,6 +45,12 @@ enum class CountryCode : uint8_t
     UNKNOWN, ///< unknown country code
 };
 
+enum WlanConnectResult
+{
+    OK = 0, ///< success
```

Comment:
consider `CONNECT_OK`. I'm worried that some badly written C headers (like in mingw or some embedded middlewares) might have a `#define` for OK.